import/PopupStackView.qml

```diff
@@ -0,0 +1,195 @@
+/*******************************************************************************
+ *  Copyright 2012-2018 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+
+import QtQuick 2.11
+import QtQuick.Controls 2.4
+import QtQuick.Layouts 1.12
+
+/*!
+ * \qmltype PopupStackView
+ * \ingroup ArcGISQtToolkit
+ * \inqmlmodule Esri.ArcGISRuntime.Toolkit
+ * \since Esri.ArcGISRutime 100.8
```

Comment:
```suggestion
 * \since Esri.ArcGISRuntime 100.8
```

This is kind of interesting doc wise. I _think_ this file is shared between C++ and QML? If so, should it be `Esri::ArcGISRuntime 100.8` or the qml version?

Reply:
I would stick to the QML versioning in QML files.

Though now that you mention it, we have  `\inqmlmodule Esri.ArcGISRuntime.Toolkit`
but we import via `import "qrc:://Esri/ArcGISRuntime/Toolkit"` syntax. I wonder if that is wrong?